core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -569,10 +581,31 @@ export class NavigationComponent implements OnInit {
               );
         this.displayParticularWorkflowVersion = displayVersionFlag;
       });
+    this.workflowExecutionService
+      .getDisplayParticularExecutionStream()
+      .pipe(untilDestroyed(this))
+      .subscribe(displayExecutionFlag => {
+        this.particularVersionDate =
+          this.execution?.startingTime === undefined
```

Comment:
you can just reuse the execution in `workspaceComponent`

Reply:
refactored